### ISSUE TITLE
add query for bestknowndigest

### ISSUE THREAD

**Issue by prestwich** [summa-tx/relays#50]

**Comment by emaG3m** [comment]
```
// GetCmdGetLastReorgLCA returns the CLI command struct for GetLastReorgLCA
func GetCmdGetLastReorgLCA(queryRoute string, cdc *codec.Codec) *cobra.Command {
	return &cobra.Command{
		Use:     "getlastreorglca",
		Example: "getlastreorglca",
		Long:    "Returns the best known digest in the relay",
```
Long description indicates getlastreorglca is the best known digest --^

**Comment by prestwich** [comment]
That is a strange error 🤔

**Comment by prestwich** [comment]
closed by #54
